### DISCUSSION TITLE
Upgrade guide (from lunarphp/admin ^0.6.2)

### DISCUSSION THREAD

**Discussion by samhk222** [lunarphp/lunar#1825] [Q&A]
Sorry for the question, but I have a problem in our project, as lunarphp/admin no longer exists, and as a result I can no longer run `composer update`

What is the step to replace the packages correctly? (I didn't find an update guide on the website either)

    _- Root composer.json requires lunarphp/admin ^0.6.2, found lunarphp/admin[0.6.4] in the lock file but not in remote repositories, make sure you avoid updating this package to keep the one from the lock file._

**Comment by glennjacobs** [comment]
Oh, you should install `lunarphp/lunar`, that is the same thing.
